packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -77,6 +77,13 @@ export enum SmsEventStatusEnum {
   CREATED = 'created',
   DELIVERED = 'delivered',
   RECEIVED = 'received',
+  ACCEPTED = 'accepted',
```

Comment:
@scopsy, @p-fernandez and @ainouzgali what do you guys think about that we add more statuses?

Reply:
I think that this is OK to have the level of verbosity. We can always decide what we are displaying in our UI. For example, we might want to simplify this to only 3 statuses and discard all the rest.

Reply:
Maybe a good move would be to have more statuses, then add an own enum of statuses of the provider and then have a mapper to make the equivalence between external and internal statutes. I suggested that in another procider webhook PR.

Reply:
Hi @p-fernandez @scopsy @davidsoderberg 
What would be the ideal thing to be done here?

I would love to follow up on the discussion you guys did and work accordingly 🙏

Reply:
@TheLearneer your implementation is fine. Due the different providers having a different set of status, the core team needs to decide which status provide utility inside of Novu, decide on them, and then map the equivalence between provider status and Novu status in the same way as you did in https://github.com/novuhq/novu/blob/d8c6b99471da2f789035eb967abee59c482f8148/providers/twilio/src/lib/twilio.provider.ts#L78

Reply:
Sorry, but I am a little confused now.
Does that means, I don't need to make any changes to the StatusEnum by myself?

Reply:
Not right now and not for this PR, as the core team has pending the work on deciding which status are going to be finally supported in Novu. Do not worry about your PR. 👍🏻